outlines/generate/choice.py

```diff
@@ -13,7 +13,11 @@ def choice(
     model, choices: List[str], sampler: Sampler = multinomial()
 ) -> SequenceGenerator:
     regex_str = r"(" + r"|".join(choices) + r")"
-    return regex(model, regex_str, sampler)
+
+    generator = regex(model, regex_str, sampler)
+    generator.format_sequence = lambda x: x
```

Comment:
This line is unnecessary, it's what the default `format_sequence` already does.

Reply:
That's what I thought as well. However monkey patching has the downside that when working in some context the patched method persists unless you reinitialize. So without the trivial lambda the integration tests will fail because it uses the format_sequence method from a prior test.